src/Generation/Generator/Renderer/Public/Class/ClassConstructors.cs

```diff
@@ -94,8 +94,14 @@ private static string RenderCallStatement(GirModel.Class cls, GirModel.Construct
         var call = new StringBuilder();
         call.Append($"var {variableName} = Internal.{cls.Name}.{Constructor.GetName(constructor)}(");
         call.Append(string.Join(", ", parameters.Select(x => x.GetCallName())));
+        if (constructor.Throws)
```

Comment:
I think we can omit the `if`? In the internal namespace the renderer is called unconditionally, too. In this way the whole logic what to print is part of `Error` which is simpler.